vyper/semantics/analysis/imports.py

```diff
@@ -80,6 +80,9 @@ def __init__(self, input_bundle: InputBundle, graph: _ImportGraph):
         self.seen: set[int] = set()
 
         self.integrity_sum = None
+        self.abosulte_search_paths = (
+            input_bundle.search_paths.copy()
+        )  # should be all system paths + topmost module path
```

Comment:
multiline expression
```suggestion
        # should be all system paths + topmost module path
        self.absolute_search_paths = input_bundle.search_paths.copy()
```